portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/liferayfaces/BridgeDemos.testcase

```diff
@@ -979,6 +979,55 @@ definition {
 			state = "FL");
 	}
 
+	@description = "This is a usecase for LPS-159239. Verify a 3.0 portlet with cdi can be deployed to a content page."
+	@priority = 3
+	test PrimeFacesCDIApplicantPortletContentPageSmoke {
+		property liferay.faces = "false";
+		property test.liferay.virtual.instance = "false";
+		property test.run.type = "single";
+
+		var facesBridgeImplDir = PropsUtil.get("liferay.faces.bridge.impl.dir");
+
+		var portletDir = "${facesBridgeImplDir}/demo/primefaces-applicant-portlet";
+
+		FileUtil.replaceStringInFile("${portletDir}/pom.xml", "4.3.0", "6.0.0");
+
+		FileUtil.replaceStringInFile("${portletDir}/src/main/webapp/WEB-INF/portlet.xml", "http://java.sun.com/xml/ns/portlet/portlet-app_2_0.xsd", "http://xmlns.jcp.org/xml/ns/portlet");
+
+		FileUtil.replaceStringInFile("${portletDir}/src/main/webapp/WEB-INF/portlet.xml", "2.0", "3.0");
```

Comment:
I assume this section at the top is upgrading a portlet to 3.0? I would recommend adding tasks or a wrapper macro here to make it more readable to other testers with less context